includes/class-freemius.php

```diff
@@ -24102,7 +24102,7 @@ function _add_trial_notice() {
 
             if ( $this->is_registered() ) {
                 // If opted-in, override trial with up to date data from API.
-                $trial_plans       = FS_Plan_Manager::instance()->get_trial_plans( $this->_plans );
+                $trial_plans       = FS_Plan_Manager::instance()->get_visible_trial_plans( $this->_plans );

```

Comment:
@DanieleAlessandra I am debating whether to replace all `get_trial_plans` call with the `get_visible_trial_plans` call. But I think at this stage, we are safe to replace just here.

Reply:
I assumed that `get_trial_plans` could be useful in some cases, not knowing the whole project I added my own function leaving other parts as they where.